storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,36 @@
+[id="persistent-storage-csi-azure"]
+= Azure CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk.
+
+:FeatureName: Azure Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
+
+???need info on how to install???To create CSI-provisioned PVs that mount to Azure storage assets, you must manually enable the Azure CSI Driver Operator and the Azure CSI driver for deployment.
+After you enable the deployment, {product-title} installs the operator, driver, and necessary sidecars.
+
+
+* The _Azure CSI Driver Operator_ provides a StorageClass by default that you can use to create PVCs. You also have the option to create the Azure StorageClass as described in xref:../../storage/persistent_storage/persistent-storage-azure.adoc#persistent-storage-using-azure[Persistent storage using Azure].
+
+* The _Azure CSI driver_ enables you to create and mount Azure PVs.
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision Azure storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.
```

Comment:
Just to confirm, [Azure Disk migration](https://issues.redhat.com/browse/STOR-576) is not targeted for 4.8, correct?

Reply:
Correct. Only Cinder and AWS EBS are in scope for migration in 4.8.